discourse.lua

```diff
@@ -67,13 +67,18 @@ app:get('/api/v1/discourse-sso', capture_errors(function(self)
     local request_payload = extract_payload(payload)
     local user = Users:select('where username = ? limit 1',
             self.session.username,
-            { fields = 'id, username, verified, role, email' })[1]
-    local response_payload = build_payload(user, request_payload.nonce)
+            { fields = 'id, username, verified, role, email, unique_email' })[1]
+
+    if user:cannot_access_forum() then
+        yield_error({ msg = 'Banned users cannot use the forum.', status = 403 })
```

Comment:
When we implement student account we should probably put this in user.role, but for now this works. 
Also, if this is returning a JSON status, we probably want a real way to handle front-end errors at some point,